package.json

```diff
@@ -14,7 +14,7 @@
     "jsonwebtoken": "^8.5.1",
     "polyfill-library": "^3.107.1",
     "serve-static": "^1.14.1",
-    "socket.io": "^4",
+    "socket.io": "^4.6.1",
```

Comment:
This does not seem to be linked to the current PR. You can open a distinct pr to update dependencies